package.json

```diff
@@ -49,7 +49,7 @@
     "@types/jest": "^27.5.0",
     "@types/lodash": "^4.14.168",
     "@types/node": "^12.12.37",
-    "@types/uuid": "^8.0.0",
+    "@types/uuid": "^7.0.8",
```

Comment:
Since we're only using `v4` (and were using a mismatched version before), I think this would have been fine. But figured it was probably best to keep these versions closely aligned, just in case.